src/pmpo_c.cpp

```diff
@@ -562,6 +588,39 @@ void polympo_getMeshVtxRotLat_f(MPMesh_ptr p_mpmesh, const int nVertices, double
   }
 }
 
+void polympo_setMeshVel_f(MPMesh_ptr p_mpmesh, const int nComps, const int nVertices, const double* array){
```

Comment:
replicate similar to: polympo_setMeshVtxCoords_f

Note: there is an assert on nVertices (so add `PMT_ALWAYS_ASSERT(p_mesh->getNumVertices()==nVertices);`), and Kokkos::create_mirror_view is used

Reply:
https://github.com/SCOREC/polyMPO/pull/51/commits/039c02801da121462b1dcd9f423f44a548647229